src/functions.php

```diff
@@ -344,3 +346,44 @@ function _current_time()
 {
     return function_exists('hrtime') ? hrtime(true) / 1e9 : microtime(true);
 }
+
+
+/**
+ * @param int $options
+ * @return UriInterface
+ * @internal
+ */
+function _idn_uri_convert(UriInterface $uri, $options = IDNA_DEFAULT)
```

Comment:
What do you think about changing `IDNA_DEFAULT` constant to `0` here? I'm think when `intl` is not available it can cause an error...

See roundcube/roundcubemail#6244 for details, also related to #2448.

Reply:
Done.